cypress/integration/conditions/numericConditions.cy.js

```diff
@@ -233,7 +233,7 @@ describe('number conditions', () => {
     })
 })
 
-describe('integer', () => {
+describe('integer', { testIsolation: false }, () => {
```

Comment:
Could you add a comment in the PR description about why these tests needed testIsolation: false ?
That info could help when upgrading our other repos

Reply:
Good comment, thanks. This actually led me to do three things:
- Add an explanation to the PR description as you requested
- Add some comments to the codebase where it is used
- Review the usage of `testIsolation: false` and I think I added it in a few places where it _should not_ be required.

So I removed those occurrences in https://github.com/dhis2/line-listing-app/pull/374/commits/e05799240178bb82e9ac85b3e26251296b68ccac and if the tests for that commit pass, I will keep things like that.